scripts/verify-phase-52-7-1-namespace-layout-inventory-contract.sh

```
@@ -0,0 +1,241 @@
+  echo "Phase 52.7.1 file movement rejected: missing current package path control-plane/aegisops_control_plane/" >&2
+  exit 1
+fi
+
+for required_path in \
+  "${repo_root}/control-plane/main.py" \
+  "${repo_root}/.github/workflows/ci.yml" \
+  "${repo_root}/scripts/verify-phase-52-7-1-namespace-layout-inventory-contract.sh" \
+  "${repo_root}/scripts/test-verify-phase-52-7-1-namespace-layout-inventory-contract.sh"
+do
+  if [[ ! -f "${required_path}" ]]; then
+    echo "Phase 52.7.1 file movement rejected: missing current path ${required_path#${repo_root}/}" >&2
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)